### ISSUE TITLE
Wyoming not found in .lookup

### ISSUE THREAD

**Issue by samfaries** [unitedstates/python-us#45]
us.states.lookup("Wyoming") returns no output. 

us.states.lookup("WY") returns the correct output.

**Comment by noahlee826** [comment]
Looks like current version (0.7.2) of Jellyfish calculates the metaphone code of "Wyoming" as "WYMNK" but the states pickle file lists the metaphone code as "YMNK".

**Comment by maxamante** [comment]
The same behavior exists for 'Delaware', 'Wisconsin', 'Washington', and 'West Virginia'.

**Comment by BraunAvi** [comment]
To add to the confusion: 
usa.states.lookup('Wayoming') returns the correct value.
(Note the wrong spelling of WAyoming)

**Comment by jhoh10** [comment]
can this be fixed?

**Comment by jcarbaugh** [comment]
I'll get a fix out for this over this upcoming weekend!

**Comment by nupanick** [comment]
Is this still being worked on? Can we get a release please?

**Comment by starlabs007** [comment]
This seems to be fixed in latest? (us=`1.0.0`, jellyfish=`0.5.6`).
I've tried Wyoming, Delaware, Wisconsin, Washington and West Virginia and they all return correct states and abbreviations.

**Comment by mstellon** [comment]
Having the same issue with latest version that I just installed (us==1.0.0, jellyfish==0.5.6)

**Comment by Jacob-Barhak** [comment]
Delaware still does not return the correct answer. I isolated the problem to Jellyfish 0.5.6 that seems to be installed by default. 

Upgrading Jellyfish to 0.6.1 solves the problem - for those who want a solution without a new release. 

It would be nice if the next version will have the correct jellyfish dependency installed by default.

**Comment by jcarbaugh** [comment]
I just pushed a big update to [a branch](https://github.com/unitedstates/python-us/tree/us2) to prepare for a 2.0 release. It includes an update to jellyfish version (to 0.7.2) and tests to make sure that lookup finds the correct state for both name and abbreviation. I've also removed the sqlite/pickle storage of states info and made it pure Python, making it easier to manage data changes.

All tests are passing, but I'd really appreciate if anyone could check out this branch and make sure it works well with their code. Thanks!
